prime/infra/dev/prime-houston-api.yaml

```diff
@@ -250,6 +250,42 @@ paths:
           description: "The customerId of the customer"
           required: true
           type: string
+  "/support/simprofile/{id}":
```

Comment:
This way of REST API URL gives a wrong impression that the "id" is Sim Profile ID.
It is different from the way Customer Endpoint API URLs are defined.
Over there, we have used this as a guideline - https://cloud.google.com/apis/design/resources

But this is not introduced in this PR.  It is for the whole support-api swagger spec.
So, I think we should do it in some other PR and let this PR merge.